docker/tests/Templates/fxdependent-centos7/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:7
+
+ARG PACKAGENAME
+ARG PACKAGELOCATION
+ARG PREVIEWSUFFIX=
+ARG TESTLIST=/PowerShell/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1,/PowerShell/test/powershell/engine/Module
+ARG TESTDOWNLOADCOMMAND="git clone --recursive https://github.com/PowerShell/PowerShell.git"
+
+# Install dependencies
+RUN yum install -y \
+        curl \
+        glibc-locale-source \
+        git
+
+# Install dotnet-runtime
+RUN rpm -Uvh https://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
+RUN yum install -y \
```

Comment:
This wont be needed if using the 2.1-sdk or  2.1-runtime image is used.

Reply:
Updated base image for ubuntu18.04 and debian.9;
Unfortunately .NET doesn't have any other images.